core-default/src/main/java/me/aurium/beetle/defaults/GenericBeetleFactory.java

```diff
@@ -33,7 +36,7 @@ public GenericBeetleFactory(String appName, Path baseFolder, boolean isDebug) {
 
     @Override
     public CommonBeetle build() {
-        Tasker tasker = new GenericTasker();
+        Tasker tasker = new GenericTasker(Executors.newFixedThreadPool(10000));
```

Comment:
Woah, woah, woah. 10,000 threads? 10,000 threads! How that would burst into a spectacular application crash!

You would rival herobrine's 3,800 timer threads! Like him, you neither use a ThreadFactory, so those thousands of threads look anonymous having generic names.

Reply:
What's a good number of threads to use? I don't see the point of having named threads.

Reply:
You'll see the point of having named threads when you need to read a thread dump. Having anonymous threads is simply not good for the organization of the software running on your JVM

Reply:
Hah! Let the api implementer deal with it, since you have to specify your thread pool if you want to make a common beetle!